src/get-started/java-binding.md

```diff
@@ -75,10 +75,9 @@ requested at any time, while `Fork` – only when the transaction is executed. T
 lifetime of these objects is limited by the scope of the method to which they
 are passed to.
 
-Exonum stores elements in the collections as byte arrays. Therefore, a user
-must implement serialization of values stored in the collection. Java Binding
-provides *serializers* for standard and some commonly used types, see
-[`StandardSerializers`][standardserializers] for details.
+Exonum stores elements in collections as byte arrays. Therefore, a
+serialization for values stored in collections must be provided.
```

Comment:
serializers for values … must be provided? (as "a serialization … must be provided" sound a little odd)